scss/_settings_grid.scss

```diff
@@ -4,15 +4,22 @@
 // Global grid settings
 $grid-max-width: 80rem !default;
 $grid-columns-small: 4 !default;
+$grid-8-columns-small: $grid-columns-small !default;
```

Comment:
Please group them by grid (new / old). And don't make new one rely on value of old variable (just hardcode correct value of 4).